service/resource/legacy/metric.go

```diff
@@ -0,0 +1,25 @@
+package legacy
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	PrometheusNamespace            = "flannel_operator"
+	PrometheusSubsystem            = "flanneld_resource"
+	VersionBundleVersionAnnotation = "giantswarm.io/version-bundle-version"
+)
+
+var versionBundleVersionGauge = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Namespace: PrometheusNamespace,
+		Subsystem: PrometheusSubsystem,
+		Name:      "version_bundle_version_total",
+		Help:      "A metric labeled by major, minor and patch version of the version bundle being in use.",
+	},
+	[]string{"major", "minor", "patch"},
```

Comment:
Why we just don't specify the name label instead? What is the benefit of having them split?

Reply:
What do you mean with name label? The reason I did it that way is to be able to operator on numbers. We can do match and comparisons against major, minor and patch versions if needed. It is more open and flexible as having a string which we would need to parse somehow.

Reply:
Good enough reason for me.